### PR TITLE
[WIP] Fix #1936 : Test for Admin Action

### LINKED ISSUE #1936: Missing Tests for Custom Admin Action in Jobs App
Writing Unit tests for the Admin sectin of the `Jobs` app

### PR DESCRIPTION
**Pull request by Ayukha**
Fixes #1936

### CODE REVIEW

**Comment by Ayukha** [comment]
@RishabhJain2018  Please review

**Comment by Ayukha** [comment]
@RishabhJain2018  Please review.

**Comment by RishabhJain2018** [comment]
@Ayukha I have already reviewed it. Please check the comments.

File: tests/unit/jobs/test_admin.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        self.user4 = User.objects.create_superuser(
+            username='test',
+            email='abc@xyz.com',
+            password='test',
+        )
+
+        self.app_admin = SubmissionAdmin(Submission, AdminSite())
+
+    def test_submit_job_to_worker(self):
+        queryset = Submission.objects.filter(status='Finished')
```
[the remark this answers is not in the document]
Comment (Ayukha) [reply tests/unit/jobs/test_admin.py]: @RishabhJain2018  Done the suggested changes.

**Comment by RishabhJain2018** [comment]
@Ayukha Any update on this?

**Comment by Ayukha** [comment]
@RishabhJain2018  I have updated the PR in my last commit. Please lt me know if any other changes are required.

File: tests/unit/jobs/test_admin.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+                challenge=self.challenge,
+                test_annotation=SimpleUploadedFile('test_sample_file.txt',
+                                                   b'Dummy file content', content_type='text/plain')
+            )
+
+        self.submission = Submission.objects.create(
+            participant_team=self.participant_team,
+            challenge_phase=self.challenge_phase,
+            created_by=self.challenge_host_team.created_by,
+            status='Finished',
```
[the remark this answers is not in the document]
Comment (Ayukha) [reply tests/unit/jobs/test_admin.py]: @RishabhJain2018  Please review.

**Comment by deshraj** [comment]
@RishabhJain2018 can you please catch up on this PR quickly?

**Pull request merged**